src/node_lib.h

```diff
@@ -11,9 +11,15 @@ namespace node { namespace lib {
 
     namespace { // private variables
         bool _event_loop_running = false;
+        v8::Isolate* _isolate = nullptr;
+        Environment* _environment = nullptr;
     }
 
-    bool EventLoopIsRunning() { return _event_loop_running; }
+    const bool EventLoopIsRunning() { return _event_loop_running; }
+
+    const v8::Isolate* Isolate() { return _isolate; }

```

Comment:
Just curious, is there a reason you did not name this getIsolate()?

Reply:
Looks like `isolate` would be correct: https://github.com/nodejs/node/blob/master/CPP_STYLE_GUIDE.md#camelcase-for-methods-functions-and-classes